invenio_rdm_records/records/jsonschemas/records/parent-v2.0.0.json

```diff
@@ -67,6 +73,38 @@
               }
             }
           }
+        },
+
+        "settings": {
```

Comment:
These access `settings` are intended to store configuration per record about who is able to request access (users/guests/both/nobody), the display text outlining the access conditions, and default values for the access request.

Right now, this PR does not include a REST API endpoint to change these values for a record.
IMO, that endpoint can come later, as it's not strictly necessary for the functioning of the access requests.